app/models/upload_file.rb

```diff
@@ -0,0 +1,72 @@
+# app/models/upload_file.rb
+class UploadFile < Item
```

Comment:
It should be "FileUpload", not "UploadFile"!  Classes are supposed to be named after nouns (or noun phrases), not verb phrases.